src/i18n/zh-cn/ui.ts

```diff
@@ -1,47 +1,64 @@
-import { UIDictionary } from '../translation-checkers';
-
-export default UIDictionary({
-    'a11y.skipLink': '跳转到内容',
-	'navbar.a11yTitle': '导航顶部',
-    // Site settings
+export default {
+	'a11y.skipLink': '跳转至内容',
+	'navbar.a11yTitle': '顶部',
+	// Site settings
 	'site.title': 'Astro 文档',
-	'site.description': '用更少的客户端 JavaScript 构建更快的站点。',
+	'site.description': '使用更少的客户端 Javascript 构建更快的网站。',
 	'site.og.imageSrc': '/default-og-image.png?v=1',
-	'site.og.imageAlt': '星辰浩瀚的太空中的 astro 标志，右侧前景中漂浮着一颗紫色的土星状行星。',
+	'site.og.imageAlt': '漫漫星空中的 Astro 图标，右侧前景中漂浮着紫色土星一般的星球',
 	// Left Sidebar
-	'leftSidebar.a11yTitle': '站点导航',
+	'leftSidebar.a11yTitle': '主要内容',
```

Comment:
```suggestion
	'leftSidebar.a11yTitle': '站点导航',
```

This is a contextualization. Combined with the accessibility feature tree, we may see that it is not the main content(主要内容), but the navigation (站点导航).

![image](https://user-images.githubusercontent.com/45708948/185060952-807188cc-cefa-4f18-9d5a-5e8bfa1c2a1e.png)